package.json

```diff
@@ -33,7 +33,8 @@
     "chalk": "1.1.3",
     "coveralls": "2.13.1",
     "ember-ajax": "2.5.6",
-    "ember-bootstrap": "^1.0.0-rc.2",
+    "bootstrap-sass": "^3.3.7",
```

Comment:
Minor; make sure your inserting the imports alphabetically. A few of these are in the wrong spots.